### PR TITLE
Add bundled configuration for Cypress

### LINKED ISSUE #311: docs: Simplify docs for cypress testing
Our Cypress testing docs can be simplified by adding Cypress to our bundled configurations. We could then remove the sections describing manual setup.

Related to https://github.com/bridgetownrb/bridgetown/issues/309#issuecomment-844974463

### PR DESCRIPTION
**Pull request by ayushn21**
This is a 🙋 feature or enhancement.

## Summary

This PR adds a bundled configuration for Cypress based on the automation by @ParamagicDev (https://github.com/ParamagicDev/bridgetown-automation-cypress).

There's an issue with running the tests though. We run `yarn start` before running the tests which forks off processes to run the Bridgetown server and Webpack dev server. These processes aren't killed after the test suite ends so the only way to run the server again is to find the process id and kill the process before running `yarn start` again.

Not quite sure how to fix this. Once we switch to Rake, would be possible to fix this problem? Might be worth holding off on merging until then.

## To Do

- [x] Figure out a fix for the the orphaned server processes during testing
- [x] Update docs on website

## Context

Resolves #311

### CODE REVIEW

**Comment by jaredcwhite** [comment]
Not sure the switch to Rake would automatically resolve the issue, but yeah would make sense to implement then and not now (thus having to reimplement later).

**Comment by ayushn21** [comment]
I took a (very) short peek at your Rake/Roda etc PR and saw that the commands in Rake were triggered using Ruby (`Bridgetown::Commands::Build.start` etc etc) ... so figured that might give us an easier way to kill the forked processes once the tests are run. This is a complete guess though; I know next to nothing about Rake and orchestrating processes and things.

**Review by jaredcwhite — APPROVED** [review]
This look great! I'll test it out more when I get a chance but meanwhile LGTM. (Just one question about the Rake tasks)

File: bridgetown-core/lib/bridgetown-core/configurations/cypress/cypress_tasks [hunk 1]
```diff
@@ -0,0 +1,25 @@
+namespace :cy do
+  task :open do
+    system "yarn run cypress open"
+  end
+
+  task :test do
+    ENV["BRIDGETOWN_ENV"] = "test"
+    server_pid = fork { Bridgetown::Commands::Start.start }
+    system "yarn run cypress open"
```
[the remark this answers is not in the document]
Comment (ayushn21) [reply bridgetown-core/lib/bridgetown-core/configurations/cypress/cypress_tasks:9]: Yeah good call! I'll update.

**Comment by ayushn21** [comment]
Thanks! I'll leave this open until you get a chance to test it, just in case something's not quite right. Seemed ok in my testing but you never know!

**Pull request merged**